src/olympia/reviews/views.py

```diff
@@ -166,7 +166,11 @@ def delete(request, addon, review_id):
     review = get_object_or_404(Review.objects, pk=review_id, addon=addon)
     if not user_can_delete_review(request, review):
         raise PermissionDenied
-    review.delete(user_responsible=request.user)
+    # Log deleting moderated reviews
+    if review.editorreview:
+        review.moderator_delete(user=request.user)
```

Comment:
For the API test, see https://github.com/mozilla/addons-server/pull/4983/files#diff-14fce06dc10fad51d5a20ae999777aa3R1215